actions/cloud.tsx

```diff
@@ -89,23 +89,29 @@ export function subscribeCloudSubscription(productId: string) {
     };
 }
 
-export function requestCloudTrial(page: string, email = '') {
+export function requestCloudTrial(page: string, subscriptionId: string, email = ''): ActionFunc {
     trackEvent('api', 'api_request_cloud_trial_license', {from_page: page});
-    return async () => {
+    return async (dispatch: DispatchFunc): Promise<any> => {
         try {
-            await Client4.requestCloudTrial(email);
+            const updatedSubscription = await Client4.requestCloudTrial(subscriptionId, email);
+            if (updatedSubscription) {
+                dispatch({
+                    type: CloudTypes.RECEIVED_CLOUD_SUBSCRIPTION,
+                    data: updatedSubscription,
+                });
+            }
         } catch (error) {
             return false;
         }
         return true;
     };
 }
 
-export function validateBusinessEmail() {
+export function validateBusinessEmail(email = '') {
     trackEvent('api', 'api_validate_business_email');
     return async () => {
         try {
-            await Client4.validateBusinessEmail();
+            await Client4.validateBusinessEmail(email);
```

Comment:
Yes, important to send the email to be validated 🙃

Reply:
See comment on the server change related to this: https://github.com/mattermost/mattermost-server/pull/20352#discussion_r898627544